docs/guides/high-availability/examples/generated/ispn-site-a.yaml

```diff
@@ -138,12 +138,14 @@ spec:
       mode: "SYNC"
       owners: "2"
       statistics: "true"
+      remoteTimeout: 14000
       stateTransfer:
         chunkSize: 16
       backups:
         site-b: # <2>
           backup:
             strategy: "SYNC" # <3>
+            timeout: 13000
```

Comment:
Should we include these timeouts in official docs? 

These were [included](https://github.com/keycloak/keycloak-benchmark/pull/681) this January so not all perf tests were done using these options. Also, if I remember correctly @pruivo mentioned it may be worth documenting these in some tuning guide rather than mentioning these directly in docs. https://github.com/keycloak/keycloak-benchmark/issues/708

Reply:
We're running our daily tests with those parameters, so it would be honest to include this setting. And yes, a tuning guide would be the next good thing. 

So until we can reference it in a tuning guide, I'd argue to keep it as is.